src/Auth0.php

```diff
@@ -306,31 +312,16 @@ public function __construct(array $config)
             $this->skipUserinfo = $config['skip_userinfo'];
         }
 
-        // If a token algorithm is passed, make sure it's a specific string.
-        if (! empty($config['id_token_alg'])) {
-            if (! in_array( $config['id_token_alg'], ['HS256', 'RS256'] )) {
-                throw new CoreException('Invalid id_token_alg; must be "HS256" or "RS256"');
-            }
-
-            $this->idTokenAlg = $config['id_token_alg'];
+        $this->idTokenAlg = $config['id_token_alg'] ?? 'RS256';
+        if (! in_array( $this->idTokenAlg, ['HS256', 'RS256'] )) {
+            throw new CoreException('Invalid id_token_alg; must be "HS256" or "RS256"');
         }
 
-        // If a token audience is passed, make sure it's an array.
-        if (! empty($config['id_token_aud'])) {
-            if (! is_array( $config['id_token_aud'] )) {
-                throw new CoreException('Invalid id_token_aud; must be an array of string values');
-            }
+        $this->idTokenIss = $config['id_token_iss'] ?? 'https://'.$this->domain.'/';
```

Comment:
Because it was before :) 

Is there any valid reason that the issuer used for the authorize endpoint would be different than the one used in the ID token? I can't think of one ...